src/index.mjs

```diff
@@ -75,13 +82,12 @@ export async function initializeVisualization(options = {}) {
         <div style="height:300px; width: 100%;" class="canvas-container hidden">
           <div id="loading-container-${indicator.id}" class="loading-overlay hidden"></div>
           <canvas id="chart-${indicator.id}"></canvas>
-          <p class="help">k: millier, M: million, Md: milliard</p>
+          ${indicator.ignoreFormatBigNumber !== true ? '<p class="help">k: millier, M: million, Md: milliard</p>' : ''}
```

Comment:
On se retrouve avec une double négation "not ignore".
Inverser pour `formatBigNumber` ou `enableFormatBigNumber` avec défaut à `true` ?
C'est aussi plus cohérent avec `enable_visualization` (toujours enable plutôt que enable/disable selon les variables).